### PR TITLE
Enable apple ci

### PR DESCRIPTION
**Pull request by bramtayl**
Just an experiment to enable CI on apple

### CODE REVIEW

**Comment by stekyne** [comment]
That's great you got around the flex/bison issue. What else is left to get this working?

**Comment by bramtayl** [comment]
I think it's probably good to merge as is. It would be good to get the disabled test working on apple at some point though

File: tests/c/cscore_tests.cpp [hunk 1]
```diff
@@ -35,17 +35,18 @@ void cscore(CSOUND *cs)
     cscoreListGetUntil(cs, 100);
 }
 
-TEST (CScoreTests, testCscore)
-{
-    csoundSetGlobalEnv("OPCODE6DIR64", "../../");
-    CSOUND *csound = csoundCreate(0);
+// TODO: fix on apple
```
Comment (stekyne) [inline tests/c/cscore_tests.cpp:38]: What needs to be fixed?

Comment (bramtayl) [reply tests/c/cscore_tests.cpp:38]: It didn't pass on CI. Don't have an apple computer so can't debug

**Comment by stekyne** [comment]
Nice work thanks

**Pull request merged**